src/locales/fr.ts

```diff
@@ -70,7 +70,7 @@ export default {
     baptism_of_the_lord: 'Baptême du Seigneur',
     christ_the_king_sunday: 'Christ, Roi de l’Univers',
     christmas: 'Nativité du Seigneur',
-    corpus_christi: 'Saint-Sacrement',
+    corpus_christi: 'Saint Sacrement',
```

Comment:
I’ll leave it up to @emagnier to decide which form to use, however, Lectionaire III uses the hyphenated form.

Reply:
For info: normally hyphens in French are only used for a place or a building, and also in the case of a few word contractions.

For example, here is the name of a town in France: "Saint-Médard-en-Jalles".

But of course, we French love the exeptions! Suddenly we will find "Fête-Dieu" or "Fête Dieu", "Saint-Sacrement" or "Saint Sacrement, etc. Both writings are therefore possible a priori. On the AELF they remove the dashes. I'll put it back on my side so it will stick better to your book.

PS: we have lots of things like that in French, which are often a matter of personal preference: this is already the case with the word "scolastique", but "scholastique" is also possible and closer to the original Latin word, d 'where my correction for the latter (by the way in English you also kept the "h").

Reply:
I understand that the hyphenation is optional. There are quite similar rules in English too. :smiley:

On the spelling of St Scholastica, I agree it looks better with the `h`. Also the same spelling is used in Lectionaire III (the same thing @emagnier would share with you privately if you want to).

Reply:
> On the spelling of St Scholastica, I agree it looks better with the `h`. Also the same spelling is used in Lectionaire III (the same thing @emagnier would share with you privately if you want to).
OK.